crowbar_framework/app/assets/stylesheets/content/footer.css.scss

```diff
@@ -1,6 +1,8 @@
 footer {
+  color: #999;
+  font-size: 12px;
+  margin-top: 40px;
```

Comment:
IMHO the padding before had been totally fine. Change this only if you really add a sticky footer.